### ISSUE TITLE
Floating IP Range

### ISSUE THREAD

**Issue by tatendam** [c4urself/OpenStack-Folsom-Install-guide#8]
Hey Guys, 

First of great guide! Works flawlessly to a tee.

I have ran into an issue with floating IP's though. I can't ping the floating IP's I attach to an instance. I lose ICMP packets when I try and ping, and then when I try to ping from the compute node itself I get the message 

root@computenode:~# ping 172.16.2.2
PING 172.16.2.2 (172.16.2.2) 56(84) bytes of data.
ping: sendmsg: Operation not permitted
ping: sendmsg: Operation not permitted
ping: sendmsg: Operation not permitted
ping: sendmsg: Operation not permitted
ping: sendmsg: Operation not permitted
^C

Any idea what might be causing this? Do I have to update IPtables or something manually? Any help will be greatly appreciated.

**Comment by kiranmurari** [comment]
Hi tatendam,

Were you able to figure out about the network connectivity issue. Even I'm stuck in a similar situation. Everything goes smooth till I try to reach my instance via the floating IP.

Appreciate any inputs.

Thanks,

**Comment by tatendam** [comment]
Hi, I was able to solve my problem by opening up TCP and ICMP Ports in the Nova security groups. You should be a able to do it from Horizon under Access and Security. See Screenshot attached. Let me know if that solves it. Good Luck. 

![Capture8](https://f.cloud.github.com/assets/3299940/259801/dbe20730-8cc6-11e2-97d8-217f96fd3f6e.PNG)

**Comment by kiranmurari** [comment]
Thanks for the response.

But we have already enabled those rules in the security group. Could you confirm if your setup consisted of only 1 NIC on the controller and compute nodes.

It would helpful if you can share your nova.conf. Also could you paste the outputs of 'ip addr show' and 'ip route show' after you created an instance.

**Comment by tatendam** [comment]
Yes, I am using a 10Gig NIC in bridge mode, the only thing I can think of that might be causing this issue is the bridge created for the NIC. I think I ended up redoing the bridge if I remember correctly. Because I had created the bridge before installing nova + nova network. I don't think it liked that, so I created the bridge first and then installed nova-network. Also make sure your Floating IP Range is an IP that your DHCP can't dish out in its range. I left a wide space from the DHCP Range to reserver for floating IP's. Below is Nova config from my compute node which is almost identical to controller but verbatim, word for word to config from the guide. 

![Capture15](https://f.cloud.github.com/assets/3299940/259884/56188a36-8cc9-11e2-9393-86cc7916bfc9.PNG)

**Comment by tatendam** [comment]
I also never made any changes to my IP Tables because Nova Network is supposed to do the NAT for you.
